src/main/java/com/ait/lienzo/shared/core/types/GroupType.java

```diff
@@ -47,7 +47,7 @@ public final String getValue()
     @Override
     public boolean equals(final Object other)
     {
-        if ((other == null) || (false == (other instanceof GroupType)))
+        if (false == (other instanceof GroupType))
```

Comment:
It's possible !(x) is more idiomatic, but I generally do not use negate operators, because they are hard to SEE just scanning the code. I have certain other idiom's I use regularly, like not using ?: just because thy make the code for me easier to scan without any mis-understanding. I have found these to save me time in maintaining a large codebase by myself.